Modules/Libs/QuestieLib.lua

```diff
@@ -590,34 +573,52 @@ objectiveFontString:Hide()
 ---@param desiredWidth number @Set the desired width to wrap, default: 275
 ---@return table[] @A table of wrapped lines
 function QuestieLib:TextWrap(line, prefix, combineTrailing, splitOnDot, desiredWidth, questid)
-    if(objectiveFontString:IsVisible()) then Questie:Error("TextWrap already running... Please report this on GitHub or Discord.") end
+    if not textWrapObjectiveFontString then
+        --                    Classic                          Wotlk Classic
+        textWrapFrameObject = _G["QuestLogObjectivesText"] or _G["QuestInfoObjectivesText"]
```

Comment:
The reason i did i this way is because if we put it in root we also need to support it in the CLI, thought it was prettier to just do it within the function as it is pretty static. Or am i misunderstanding the question? :D